Makefile

```diff
@@ -55,6 +55,13 @@ else
 	rm temp-test.in
 endif
 	make install-flexmeasures
+	make install-highs
+
+install-highs:
```

Comment:
I believe new commands should be added to the PHONY list at the top. Not sure why, actually.

Reply:
Added install-highs to PHONY. "The special target .PHONY: allows to declare phony targets, so that make will not check them as actual file names: it will work all the time even if such files still exist."

Reply:
Done. "The special target .PHONY: allows to declare phony targets, so that make will not check them as actual file names: it will work all the time even if such files still exist."